zulipterminal/model.py

```diff
@@ -344,6 +347,15 @@ def update_stream_message(self, topic: str, msg_id: int,
         display_error_if_present(response, self.controller)
         return response['result'] == 'success'
 
+    def fetch_custom_emojis(self) -> Dict[str, Dict[str, str]]:
+        response = self.client.get_realm_emoji()
+        custom_emojis = {}
+        for emoji_code, emoji in response['emoji'].items():
+            if not emoji['deactivated']:
+                custom_emojis[emoji['name']] = {'code': emoji_code,
+                                                'type': 'realm_emoji'}
```

Comment:
Are these all `realm_emoji`? In reactions we have unicode/realm/zulip_extra?

Reply:
Custom emojis are all `realm_emoji`